v3/plugins/ms-kubernetes-tools/vscode-kubernetes-tools/0.1.17/meta.yaml

```diff
@@ -14,5 +14,6 @@ spec:
   containers:
     - image: "eclipse/che-remote-plugin-kubernetes-tooling-0.1.17:7.0.0-next"
       name: "vscode-kubernetes-tools"
+      memoryLimit: "1G"
```

Comment:
What was the previous default value?
`1G` looks more than is needed for toolings that communicate with K8s/OS clusters. I may be wrong )

Reply:
I checked it with ~700 mb several months ago and it was OOM Killed